### PR TITLE
Makefile for the Vim documentation on Windows

### PR DESCRIPTION
**Pull request by RestorerZ**
Preparation of documentation (not all) on Vim in Windows.

Sometimes I'm too lazy to run a UNIX VM, so I did it for myself, maybe it will be useful to someone else.

### CODE REVIEW

File: runtime/doc/vim2html.pl [hunk 1]
```diff
@@ -1,16 +1,23 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
```
Comment (chrisbra) [inline runtime/doc/vim2html.pl]: that looks wrong

File: runtime/doc/vim2html.pl [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
-$date = `date`;
-chop $date;
+# 30.11.23, Restorer:
+# This command does not work in OS Windows.
+# The "date" command in Windows is different from its counterpart in UNIX-like systems.
+# The closest analog is the "date /t" command, but how it would work in UNIX,
+# I don't know. I've corrected it as best I can. I don't know Perl.
+#$date = `date`;
+#chop $date;
+$date = localtime();
```
Comment (chrisbra) [inline runtime/doc/vim2html.pl]: Can we format the localtime() command more like a normal `date` output?

**Comment by chrisbra** [comment]
I think it is okay, except for the minor things in the perl file. 

Can you please translate the russian comments into English?

File: runtime/doc/vim2html.pl (hunk 1 above)
Comment (RestorerZ) [reply runtime/doc/vim2html.pl]: As I understand it, this is not really required for Windows OS. On the contrary, it is required for UNIX-like systems. But judging by [this comment](https://unix.stackexchange.com/questions/29608/why-is-it-better-to-use-usr-bin-env-name-instead-of-path-to-name-as-my), the attitude to "env" is not quite unambiguous.
I did it as it was originally.

File: runtime/doc/vim2html.pl (hunk 2 above)
Comment (RestorerZ) [reply runtime/doc/vim2html.pl]: `$date = localtime(); # outputs like this Fri Nov 3 00:56:59 2023`
Which, as I recall, is about the same as the "date" command with no arguments.
I made that now the date is output in the format DD.MM.YYYY.

**Comment by RestorerZ** [comment]
> Can you please translate the russian comments into English?
Done.

**Comment by chrisbra** [comment]
okay thanks!

**Pull request merged**